lib/internal/cluster/master.js

```diff
@@ -104,8 +104,18 @@ function createWorkerProcess(id, env) {
   workerEnv.NODE_UNIQUE_ID = '' + id;
 
   if (execArgv.some((arg) => arg.match(debugArgRegex))) {
-    execArgv.push(`--inspect-port=${process.debugPort + debugPortOffset}`);
-    debugPortOffset++;
+    let inspectPort;
+    if (cluster.settings.inspectPort) {
+      if (typeof cluster.settings.inspectPort === 'function')
```

Comment:
Like in https://github.com/mutantcornholio/node/blob/17a1611b402bace03b9495662c721a6a758c0eb2/lib/internal/child_process.js#L268